src/rpc/mempool.cpp

```diff
@@ -855,8 +855,8 @@ static RPCHelpMan submitpackage()
             },
         },
         RPCExamples{
-            HelpExampleCli("testmempoolaccept", "[rawtx1, rawtx2]") +
-            HelpExampleCli("submitpackage", "[rawtx1, rawtx2]")
+            HelpExampleRpc("submitpackage", R"(["rawtx1", "rawtx2"])") +
+            HelpExampleCli("submitpackage", R"(["rawtx1", "rawtx2"])")
```

Comment:
I don't really understand why  b2fb55cabc76d842c58b51ff9c64126e6639d1bb is better - I'd interpret this as an array of literal strings "rawtx1", "rawtx2" ?

Reply:
The main point of b2fb55cabc76d842c58b51ff9c64126e6639d1bb is to remove the `testmempoolaccept` reference. I use the opportunity to align the string notation with the (I think) more recent usage of raw literal string notation, which is a bit clearer imo. Using quotation marks around the "variable" name seems consistent with every other instance I've found, e.g. [`testmempoolaccept`](https://github.com/bitcoin/bitcoin/blob/7143d4388407ab3d12005e55a02d5e8f334e4dc9/src/rpc/mempool.cpp#L153), [`generateblock`](https://github.com/bitcoin/bitcoin/blob/7143d4388407ab3d12005e55a02d5e8f334e4dc9/src/rpc/mining.cpp#L318), [`combinerawtransaction`](https://github.com/bitcoin/bitcoin/blob/7143d4388407ab3d12005e55a02d5e8f334e4dc9/src/rpc/rawtransaction.cpp#L647) and [`combinepsbt`](https://github.com/bitcoin/bitcoin/blob/7143d4388407ab3d12005e55a02d5e8f334e4dc9/src/rpc/rawtransaction.cpp#L1468)

So I think these changes make sense?

Reply:
Marking as resolved for now, lmk if you still have concerns about this.

Reply:
if we're lining up ala `testmempoolaccept` shouldn't we be adding single quotes around the whole array as well?

Reply:
You're right, force pushed to add single quotes, thanks.